Assets/Scripts/Menu/MenuBird.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MenuBird : MonoBehaviour
+{
+    public Vector2 velocity = new Vector2(2.0f, 0);
+
+    private float screenEnds = 20;
+
+
+    public void Update()
+    {
+        if (transform.position.x > screenEnds) transform.position = new Vector2(-1 * screenEnds, transform.position.y);
```

Comment:
Going off comment above, if there is someway to get the viewport width, a variable could be created for both the left and right viewport limits and calculated once ahead of time.
```cs
private float screenRight = camera.viewport.width/2;
private float screenLeft = -1 * screenRight;
.
.
.
if (transform.position.x > screenRight) ...
else if (transform.position.x < screenLeft) ...
```